mtp_noms_ops/apps/security/forms/object_list.py

```diff
@@ -37,10 +37,6 @@ class SearchFormV2Mixin(forms.Form):
     # indicates whether the form was used in advanced search
     advanced = forms.BooleanField(initial=False, required=False)
 
-    def get_ga_event_category(self):
```

Comment:
@ushkarev unless I'm missing something, this method was only used to override the category property of the event sent to GA right?

e.g. was overriding this: https://github.com/ministryofjustice/django-form-error-reporting/blob/e5e41f1b3e1bd6c33f6840c10ad7dadb6ebee1c3/form_error_reporting.py#L82-L86

So it should be safe (and clean) to remove - or am I missing something here?